docs/pages/identity-governance/access-requests/automatic-reviews.mdx

```diff
@@ -90,43 +100,76 @@ does not support wildcard or regular expressions.
   by the IdP.
 </Admonition>
 
-## Step 3/4. Create automatic review rule
+## Role-based automatic reviews
 
-Next, define the automatic review rule via the Teleport Web UI.
+To demonstrate role-based automatic reviews, create the following Access Automation
```

Comment:
Instead of saying "role-based automatic reviews", let's re-iterate the use-case it addresses. Something like "To demonstrate how role requests from a specific team can be automatically approved, ..."